testapi.pm

```diff
@@ -477,11 +463,8 @@ been matched at the time of the call.
 
 =cut
 
-sub match_has_tag {
-    my ($tag) = @_;
-    if ($last_matched_needle) {
-        return $last_matched_needle->{needle}->has_tag($tag);
-    }
+sub match_has_tag ($tag) {
+    return $last_matched_needle->{needle}->has_tag($tag) if $last_matched_needle;
     return;
```

Comment:
```suggestion
    return undef;
```

Might as well fix the return value here. We should always return explicitly

Reply:
done